portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// Adds a random fun fact to the page.
+function addRandomFunFact() {
+  const funFacts =
+      ['I am a Black Belt in Mixed Martial Arts 🥋', 'I am the Internal Vice President of Women in Information and Computer Sciences at UCI 👩💻',
+      'I played trumpet in my high school marching band 🎺', 'I prefer dogs over cats, but both are equally lovable and adorable! 🐶🐱'];
```

Comment:
Lines should have a 80 column max width, so we should break this array into multiple lines. Ex:
['I am a Black Belt....',
 'I am the Internal Vice President...',
 ...]

If a single string is still too long, you can break it into separate strings. Ex:
["I am ..." +
  " .... ",
  "...."]
But you'll have to check your exact JS syntax on that since I'm not a JS expert :P 

https://google.github.io/styleguide/jsguide.html#formatting-column-limit

Reply:
Makes sense! I will also keep these style guides open, didn't realize how helpful they could be :)